frontend/public/components/events.jsx

```diff
@@ -83,207 +83,239 @@ const kindFilter = (reference, { involvedObject }) => {
+            ? resourceTypeAll
+            : [...selected].join(',')
+        }
+        mock={props.mock}
+        textFilter={textFilter}
+      />
+    </>
+  );
+};
 
-export const NoEvents = () => (
-  <Box className="co-sysevent-stream__status-box-empty">
-    <div className="text-center cos-status-box__detail">No events in the past hour</div>
-  </Box>
-);
+export const NoEvents = () => {
+  const { t } = useTranslation();
+  return (
+    <Box className="co-sysevent-stream__status-box-empty">
+      <div className="text-center cos-status-box__detail">
+        {t('events~No events in the past hour')}
+      </div>
+    </Box>
+  );
+};
 
-export const NoMatchingEvents = ({ allCount }) => (
-  <Box className="co-sysevent-stream__status-box-empty">
-    <div className="cos-status-box__title">No matching events</div>
-    <div className="text-center cos-status-box__detail">
-      {allCount}
-      {allCount >= maxMessages && '+'} events exist, but none match the current filter
-    </div>
-  </Box>
-);
+export const NoMatchingEvents = ({ allCount }) => {
+  const { t } = useTranslation();
+  return (
+    <Box className="co-sysevent-stream__status-box-empty">
+      <div className="cos-status-box__title">{t('events~No matching events')}</div>
+      <div className="text-center cos-status-box__detail">
+        {allCount >= maxMessages
+          ? t('events~{{allCount}}+ events exist, but none match the current filter', { allCount })
+          : t('events~{{allCount}} events exist, but none match the current filter', { allCount })}
+      </div>
+    </Box>
+  );
+};
 
-export const ErrorLoadingEvents = () => (
-  <Box>
-    <div className="cos-status-box__title cos-error-title">Error loading events</div>
-    <div className="cos-status-box__detail text-center">
-      An error occurred during event retrieval. Attempting to reconnect...
-    </div>
-  </Box>
-);
+export const ErrorLoadingEvents = () => {
+  const { t } = useTranslation();
+  return (
+    <Box>
+      <div className="cos-status-box__title cos-error-title">
+        {t('events~Error loading events')}
+      </div>
+      <div className="cos-status-box__detail text-center">
+        {t('events~An error occurred during event retrieval. Attempting to reconnect...')}
+      </div>
+    </Box>
+  );
+};
 
-export const EventStreamPage = withStartGuide(({ noProjectsAvailable, ...rest }) => (
-  <>
-    <Helmet>
-      <title>Events</title>
-    </Helmet>
-    <EventsList
-      {...rest}
-      autoFocus={!noProjectsAvailable}
-      mock={noProjectsAvailable}
-      title="Events"
-    />
-  </>
-));
+export const EventStreamPage = withStartGuide(({ noProjectsAvailable, ...rest }) => {
+  const { t } = useTranslation();
+  return (
+    <>
+      <Helmet>
+        <title>{t('events~Events')}</title>
+      </Helmet>
+      <EventsList
+        {...rest}
+        autoFocus={!noProjectsAvailable}
+        mock={noProjectsAvailable}
+        title={t('events~Events')}
```

Comment:
```suggestion
        title={title}
```
with additional change above.